sw/device/lib/testing/test_framework/FreeRTOSConfig.h

```diff
@@ -26,7 +26,7 @@
 #define configCHECK_FOR_STACK_OVERFLOW 1
 #define configMINIMAL_STACK_SIZE 256  // in words
 #define configSTACK_DEPTH_TYPE uint16_t
-#define configTOTAL_HEAP_SIZE ((size_t)0x800u)
+#define configTOTAL_HEAP_SIZE ((size_t)0x8000u)
```

Comment:
nit: do we need to update any linker scripts?

Reply:
No we should not need to update any linker script as the FreeRTOS heap is an uninitialized statically allocated array that we mark with a specific [section label](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/ottf_common.ld;drc=88f78ad969a50dc088008ac58f399f524288bc91;l=226). It gets placed in the bss section.